.semaphore/semaphore.yml

```diff
@@ -95,14 +81,3 @@ blocks:
             # install confluent-kafka
             - python setup.py build && python setup.py install
             - make docs
-agent:
```

Comment:
agent is required - https://confluentinc.semaphoreci.com/workflows/3b387e90-8f40-4fde-8810-7d3de6971e67?pipeline_id=fad37d9d-6874-4e43-9a19-fb9cec605df7